### DISCUSSION TITLE
Does GPTResearcher have an API

### DISCUSSION THREAD

**Discussion by donvito** [assafelovic/gpt-researcher#247] [Q&A]
Does GPTResearcher have an API like the one in Tavily? We're thinking of using it for RAG but Tavily has API limits. We're thinking to host it in our cloud. Thanks.

**Comment by notvicent3** [comment]
Hello @donvito you can send an email to support@tavily.com for questions about the API's rate limit :)

**Reply by donvito** [reply]
I just sent an email from my company email. Thanks! :)
